manifests/pipecd/values.yaml

```diff
@@ -10,6 +10,13 @@ ingress:
     # kubernetes.io/ingress.allow-http: "false"
     # kubernetes.io/ingress.global-static-ip-name: pipecd
 
+# ServiceAccount
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
```

Comment:
Could you add a `name` field to allow users to use an existing Service Account if they want?

Same as the chart for Piped Agent. https://github.com/pipe-cd/pipecd/blob/master/manifests/piped/values.yaml#L89-L93